README.md

```diff
@@ -1 +1,12 @@
 This web-application written by Python and has CRUD management: user  will be able to create new workers, read information about every worker, update this information or delete worker if he is fired. All updates wiil be written in attached database.
```

Comment:
Could you please modify this line so that the functions are displayed as a list. See Step 12 for more details.